fileutil/folders.go

```diff
@@ -23,3 +23,21 @@ func FolderExists(dir string) error {
 
 	return nil
 }
+
+// FileExists ensures that file exists
+func FileExists(file string) error {
```

Comment:
would not it be better to return boolean?

Reply:
how the app shall distinguish if it's a `folder` or `does not exist`?